Ably.podspec

```diff
@@ -21,5 +21,5 @@ Pod::Spec.new do |s|
   s.exclude_files           = 'Source/include/', 'Source/PrivateHeaders/'
   s.module_map              = 'Source/Ably.modulemap'
   s.dependency 'msgpack', '0.4.0'
-  s.dependency 'AblyDeltaCodec', '1.3.1'
+  s.dependency 'AblyDeltaCodec', '1.3.2'
```

Comment:
This sort of change should be under a discrete PR, as it's not within scope of a release PR. That's in part because this particular change is _also_ incomplete as this dependency version [is also referenced](https://github.com/ably/ably-cocoa/search?q=delta-codec-cocoa) in `Cartfile` and `Package.swift` (plus their lock / resolution files).

Reply:
Sure, this change has been moved to https://github.com/ably/ably-cocoa/pull/1217